src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,93 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_ID_LENGTH = 1;
+    private static final int MAX_ID_LENGTH = 200;
```

Comment:
The constants `MIN_ID_LENGTH` and `MAX_ID_LENGTH` are misleading as they are used for ID values, not lengths. Consider renaming them to `MIN_ID_VALUE` and `MAX_ID_VALUE` for better clarity.